lib/dbservice/utils/util.ex

```diff
@@ -6,10 +6,13 @@ defmodule Dbservice.Utils.Util do
   import Ecto.Query
 
   def invalidate_future_date(changeset, date_field_atom) do
-    today = Date.utc_today()
+    utc_now = DateTime.utc_now()
+
+    ist_now = DateTime.add(utc_now, 5 * 60 * 60 + 30 * 60, :second)
```

Comment:
Maybe use the `Timex` library instead of doing it manually? 

https://stackoverflow.com/questions/62443581/what-is-the-correct-way-of-shifting-the-timezone-elixir